android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/RpcClient.java

```diff
@@ -272,6 +272,11 @@ public synchronized boolean authorize(String password) {
         return true;
     }
 
+
```

Comment:
Why is there extra whitespace here?

Reply:
Probably added something there to test and didn't remove the line